assets/js/modules/search-console/datastore/report.js

```diff
@@ -88,37 +118,28 @@ const gatheringDataStore = createGatheringDataStore( 'search-console', {
 	dataAvailable:
 		global._googlesitekitModulesData?.[ 'data_available_search-console' ],
 	selectDataAvailability: createRegistrySelector( ( select ) => () => {
-		const rangeArgs = {
-			compare: true,
-			offsetDays: DATE_RANGE_OFFSET,
-		};
-
-		const url = select( CORE_SITE ).getCurrentEntityURL();
-		const { compareStartDate: startDate, endDate } =
-			select( CORE_USER ).getDateRangeDates( rangeArgs );
-
-		const args = {
-			dimensions: 'date',
-			startDate,
-			endDate,
-		};
-
-		if ( url ) {
-			args.url = url;
-		}
-
+		const reportArgs = getZeroDataReportArgs( select );
```

Comment:
This looks like it could be a mistake since the other instances were used within the selector for `hasZeroData` but this is used within the selector for data availability.

Really the two reports use the same args and data availability / zero data are essentially qualities of the response so it might be worth identifying a better name to use for the common args function.